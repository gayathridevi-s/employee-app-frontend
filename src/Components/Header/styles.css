.header {
    top: 0px;
    left: 0px;
    width: 100%;
    height: 77px;
    background: #FFFFFF 0% 0% no-repeat padding-box;
    padding-left: 20px;
    padding-top: 12px;
    box-shadow: rgba(99, 99, 99, 0.2) 0px 2px 8px 0px;

}

.header>img {
    top: 16px;
    left: 46px;
    width: 208px;
    height: 45px;
}