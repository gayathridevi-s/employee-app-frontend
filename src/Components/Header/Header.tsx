import { FC } from 'react';
import './styles.css';
const Header: FC = () => {
  return (
    <header className='header'>
      <img src='/assets/img/kv-logo.png' alt='kv-image' width={200}></img>
    </header>
  );
};

export default Header;
