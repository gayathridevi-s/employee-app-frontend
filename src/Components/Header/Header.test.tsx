import React from 'react';
import { render, screen } from '@testing-library/react';
import Header from './Header';

describe('Header Component', () => {
  test('renders logo image', () => {
    render(<Header />);

    const logoImage = screen.getByAltText('kv-image');

    expect(logoImage).toBeTruthy();
  });

  test('matches snapshot', () => {
    const { asFragment } = render(<Header />);

    expect(asFragment()).toMatchSnapshot();
  });
});
