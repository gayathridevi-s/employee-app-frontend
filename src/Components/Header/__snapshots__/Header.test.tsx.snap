// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component matches snapshot 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <img
      alt="kv-image"
      src="/assets/img/kv-logo.png"
      width="200"
    />
  </header>
</DocumentFragment>
`;
